### DISCUSSION TITLE
Number of pages in a pdf file

### DISCUSSION THREAD

**Discussion by mazepa-pipeline** [jsvine/pdfplumber#349] [Ask for help with specific PDFs]
Hello there.

I am python beginner. I found pdfplumber a very good tool to extract data from pdf files. I am trying to figure out how to get the number of pages in a given pdf file using pdfplumber. Anyone could give me some clues ? Thanks a lot.

**Comment by jsvine** [comment]
Hello, and wishing you luck with your Python learning journey. To get the number of pages:

```python
import pdfplumber
with pdfplumber.open("/path/to/file.pdf") as pdf:
    print(len(pdf.pages))
```

**Reply by mazepa-pipeline** [reply]
Hi jsvine ! Thank you very much for your reply. It is so simple and worked fine.
